frontend/components/NewsletterSection.js

```diff
@@ -1,20 +1,19 @@
 import React from "react";
 import styles from "../styles/Newsletter.module.css";
-import NewsletterForm from "../components/NewsletterForm";
+import NewsletterForm from "./NewsletterForm";
 import { Modal, Backdrop, Fade } from "@material-ui/core";
 import Image from "next/image";
 
 const NewsletterSection = ({ setOpen, open, callbackModal, fade }) => {
   return (
     <div id={styles.container}>
-      <div id={styles.newsletterBgContainer}>
-        <Image
-          src={`/newsletter-background.png`}
-          className={styles.background}
-          alt="banner"
-          layout={'fill'}
-        />
-      </div>
+      <Image
+        src="/newsletter-background.png"
+        className={styles.background}
+        alt="banner"
+        width="2000px"
+        height="450px"
```

Comment:
Is there a reason why a lot of changes were made to the `NewsletterForm` and `NewsletterSection` components? Seems to be causing some weird UI issues on my end 
![image](https://user-images.githubusercontent.com/80564580/177755184-811d3682-3dde-487d-b397-d85fcbed382f.png)